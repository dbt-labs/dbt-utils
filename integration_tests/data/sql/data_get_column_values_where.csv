field,condition
a,left
b,right
c,left
d,right
e,left
f,right
g,left
g,right
g,left
g,right
g,left