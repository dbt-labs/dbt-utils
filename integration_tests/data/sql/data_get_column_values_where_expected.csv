field
a
c
e
g